package.json

```diff
@@ -2,9 +2,9 @@
   "name": "jpack",
   "version": "0.0.1",
   "description": "",
-  "main": "main.js",
+  "main": "src/jpack.js",
   "scripts": {
-    "test": "npm test"
+    "test": "node ./test/lab.js"
```

Comment:
I won't accept you change, I use nokit to test. See my project `package.json` like https://github.com/ysmood/yaku